pootle/templates/admin/xhr_summary_report.html

```diff
@@ -12,10 +12,10 @@
     </thead>
     <tbody>
     <% _.each(data.summary, function (row) { %>
-      <tr class="item">
+      <tr class="item is-visible">
```

Comment:
I will rewrite the commit message, but let me explain here. When I tried to test if the `ngettext` calls worked with my changes I was unable to see the texts rendered because they are in a table with rows that use the `item` class. This class hides the row by default: https://github.com/translate/pootle/blob/be4aa70ca1545a8e3f38e0e1b28a3082b8b71062/pootle/static/css/style.css#L756-L764